app/controllers/sessions_controller.rb

```diff
@@ -2,14 +2,14 @@ class SessionsController < ApplicationController
   def new; end
 
   def create
-    session = params[:session]
-    user = User.find_by email: session[:email].downcase
-    if user&.authenticate(session[:password])
+    sessions = params[:session]
+    user = User.find_by email: sessions[:email].downcase
+    if user&.authenticate(sessions[:password])
       log_in user
-      session[:remember_me] == Settings.min1 ? remember(user) : forget(user)
+      sessions[:remember_me] == "1" ? remember(user) : forget(user)
```

Comment:
sao k dùng settings nữa e ?

Reply:
em sửa lại toàn bộ code trong ấy nên em quên chưa sửa